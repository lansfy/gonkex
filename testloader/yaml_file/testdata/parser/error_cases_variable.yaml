- name: loader MUST read tests with variables
  method: POST
  path: /some/path
  request: requestbody
  response:
    200: someresponsebody
  cases:
    - variables:
        foo1: bar1
        foo2:
          not: string_value
