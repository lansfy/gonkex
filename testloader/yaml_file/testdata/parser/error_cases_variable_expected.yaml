- Error: "unmarshal file testdata/parser/error_cases_variable.yaml: yaml: unmarshal errors:\n  line 11: cannot unmarshal !!map into string"
